core/kazoo_number_manager/src/carriers/knm_bandwidth2.erl

```diff
@@ -54,33 +55,48 @@
 
 -ifdef(TEST).
 -define(BW2_ACCOUNT_ID, "eunit_testing_account").
+-define(IS_SANDBOX_PROVISIONING_TRUE, 'true').
```

Comment:
Hmm, I would argue that's an anti-feature (compiling whole project for testing a specific application). Maybe testing can follow the .app file and `compile-test` the dependent apps (if a kazoo core/app)?